Files origsrc/libgnurdf-0.3.1/docs/api/info/libgnurdf.info and src/libgnurdf-0.3.1/docs/api/info/libgnurdf.info differ
--- origsrc/libgnurdf-0.3.1/libgnurdf/Makefile.am	2002-05-14 14:57:31.000000000 +0900
+++ src/libgnurdf-0.3.1/libgnurdf/Makefile.am	2012-04-12 18:05:10.703814200 +0900
@@ -3,8 +3,6 @@
 libincludedir = $(includedir)/libgnurdf
 
 libinclude_HEADERS = \
-	schema.h \
-	types.h \
 	alt.h \
 	attribute.h \
 	bag.h \
@@ -42,7 +40,7 @@ libgnurdf_la_SOURCES = \
 libgnurdf_la_LIBADD = $(XML_LIBS)
 
 libgnurdf_la_LDFLAGS = \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -no-undefined
 
 INCLUDES = \
 	-I$(top_srcdir) \
--- origsrc/libgnurdf-0.3.1/libgnurdf.m4.in	2001-11-03 17:06:58.000000000 +0900
+++ src/libgnurdf-0.3.1/libgnurdf.m4.in	2012-04-12 18:05:10.703814200 +0900
@@ -4,7 +4,7 @@
 dnl AM_PATH_GNURDF([MINIMUM-VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND]]])
 dnl Test for GNURDF, and define GNURDF_CFLAGS and GNURDF_LIBS
 dnl
-AC_DEFUN(AM_PATH_GNURDF,
+AC_DEFUN([AM_PATH_GNURDF],
 [dnl 
 dnl Get the cflags and libraries from the gnurdf-config script
 dnl
